content/docs/resiliency/timeouts.md

```diff
@@ -1,16 +1,27 @@
 ---
-title: Timeouts
+title: Retries and timeouts
```

Comment:
I am wondering if these two should be a separate topics? we usually have them in separate guides